content/basics/use-ably.textile

```diff
@@ -0,0 +1,139 @@
+---
+title: How to use Ably
+meta_description: "A brief introduction on the options available to use with Ably"
+---
+
+Ably provides a number of APIs, SDKs, and adaptors and this page will briefly cover these options available to help you integrate with Ably in the way best suited for you and your business. 
+
+h2(#sdks). Ably SDKs with Realtime and REST Interfaces
+
+All official client library SDKs developed by Ably offer a consistent yet idiomatic API across every language. So regardless of your development environment or language, Ably keeps things simple for you by being consistent and predictable.
+
+The Ably platform also provides a protocol adapters service providing open protocol support and interoperability with a huge array of third party client libraries supporting every imaginable platform.
+
+Ably provides "a number of SDKS":https://ably.com/download in the most popular languages.
+
+h2(#realtime-api). Ably Realtime API
```

Comment:
This should be a H3 if it's a sub-section and should be kept consistent with terminology; i.e. discussing in terms of a realtime interface.